goldenmaster/examples/olinkserver/main.cpp

```diff
@@ -1,38 +1,55 @@
 
-#include "testbed2/lib/manyparaminterface.h"
+#include "testbed2/implementation/manyparaminterface.h"

```

Comment:
one the one hand we use lib as abbreviation for library but do not use impl for implementation. I would prefer impl over implementation.